### PR TITLE
Man pages for PBS 19.4.1

### PR DESCRIPTION
**Pull request by agurban**
#### Describe Bug or Feature

PBS Professional 19.4.1 man pages

#### Describe Your Change

Updated man pages for PBS 19.4.1

### CODE REVIEW

**Comment by mike0042** [comment]
Hello @agurban! One thing remains to be done now that all of the doc file changes are made. We need to update doc/Makefile.am to include the names of the new files that were added. These files were:

- doc/man3/pbs_asyrunjob.3B
- doc/man3/pbs_confirmresv.3B
- doc/man3/pbs_preempt_jobs.3B
- doc/man3/pbs_stathost.3B
- doc/man3/pbs_statvnode.3B

Each of these needs to be added to the list of files associated with notrans_dist_man3_MANS. Thanks!

File: doc/man3/pbs_asyrunjob.3B [hunk 1]
```diff
@@ -0,0 +1,210 @@
...
+request and sends it to the server over the connection specified by 
+.I connect.  
+
+The server validates the request and replies before initiating the execution of the job.  
+
+You can use this version of the call to reduce latency in scheduling,
+especially when the scheduler must start a large number of jobs.
+
+.SH REQUIRED PRIVILEGE
+You must have Manager or Operator privilege to use this command.
```
Comment (bhroam) [inline doc/man3/pbs_asyrunjob.3B]: This is an API call, not a command (same with pbs_runjob())

File: doc/man3/pbs_asyrunjob.3B [hunk 2]
```diff
@@ -0,0 +1,210 @@
...
+
+If an error occurred, the routine returns a non-zero exit value, and
+the error number is available in the global integer 
+.I pbs_errno.
+
+.SH SEE ALSO
+qrun(1B), pbs_connect(3B), pbs_runjob(3B)
+
+
+
```
Comment (bhroam) [inline doc/man3/pbs_asyrunjob.3B]: Do you want this much blank space at the bottom?

File: doc/man3/pbs_confirmresv.3B [hunk 3]
```diff
@@ -0,0 +1,135 @@
...
+.fi
+
+.SH DESCRIPTION
+Issues a batch request to confirm a PBS advance, standing, or maintenance reservation.
+
+This function generates a 
+.I Confirm Reservation 
+(75) batch request and sends it to the server over the connection specified by 
+.I connect.
+
```
Comment (bhroam) [inline doc/man3/pbs_confirmresv.3B]: In pbs_runjob() and pbs_rerunjob() you have a second that they require privilege.  This one does as well.

File: doc/man3/pbs_holdjob.3B [hunk 4]
```diff
@@ -33,53 +33,76 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (bhroam) [inline doc/man3/pbs_holdjob.3B]: Do you want this much blank space?

File: doc/man3/pbs_preempt_jobs.3B [hunk 5]
```diff
@@ -0,0 +1,129 @@
...
+Format for a job array ID:
+.br
+.I <sequence number>[].<server name>
+.br
+For example:
+.RS 11
+.nf
+const char *joblist[3];
+joblist[0]="123@myserver";
+joblist[1]="456@myserver";
```
Comment (bhroam) [inline doc/man3/pbs_preempt_jobs.3B]: Right above you say <sequence number>.<server name> and here you are using an @.

File: doc/man3/pbs_preempt_jobs.3B [hunk 6]
```diff
@@ -0,0 +1,129 @@
...
+
+
+
+
+
+
+
+
+
+
```
Comment (bhroam) [inline doc/man3/pbs_preempt_jobs.3B]: Do you want this much blank space?

File: doc/man3/pbs_selstat.3B [hunk 7]
```diff
@@ -33,218 +33,252 @@
...
-The entry,
-.I attribs ,
-is a pointer to a list of attrl structures defined in
-pbs_ifl.h as:
-.sp
-.Ty
-.nf
-    struct attrl {
-        struct attrl *next;
+struct attrl {
```
Comment (bhroam) [inline doc/man3/pbs_selstat.3B]: This is an attropl (just like select)

File: doc/man8/pbs_sched.8B [hunk 8]
```diff
@@ -610,9 +605,9 @@ Default:
 
 .IP preempt_checkpoint 13
 .B Deprecated. 
-Add "C" to 
+Add "C" to the
```
Comment (bhroam) [inline doc/man8/pbs_sched.8B]: This isn't needed, preempt_checkpoint was part of the original NASA beta back in the late 90s.  It never made it into a real version.

**Comment by agurban** [comment]
I've filed doc bugs to address these issues.  They'll be fixed in the next version.

**Pull request merged**